Dockerfile

```diff
@@ -0,0 +1,16 @@
+# When updating check if similar changes need to be made to cmd/otelcontribcol/Dockerfile
+FROM golang:1.14 AS build
+
+WORKDIR /src
+ADD . /src
+
+RUN make otelcontribcol
+
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
+ENTRYPOINT ["/otelcontribcol"]
```

Comment:
if a config is not specified then otel will not start

Reply:
Yeah - this is a package of the binary, are you thinking we should package a default config with it too? Seems like a nice idea if we know what a good default is.

Reply:
if the goal is to make it functional then a good idea -- examples directory has a config that can be used